helperAPI.py

```diff
@@ -628,26 +636,31 @@ def maskString(string):
 
 def printHoldings(brokerObj: Brokerage, loop=None, mask=True):
     # Helper function for holdings formatting
-    printAndDiscord(
-        f"==============================\n{brokerObj.get_name()} Holdings\n==============================",
-        loop,
-    )
+    EMBED = {
+        "title": f"{brokerObj.get_name()} Holdings",
+        "color": 3447003,
+    }
+    printing = ""
+    EMBED["fields"] = []
     for key in brokerObj.get_account_numbers():
         for account in brokerObj.get_account_numbers(key):
-            printAndDiscord(f"{key} ({maskString(account) if mask else account})", loop)
+            field = {
+                "name": f"{key} ({maskString(account) if mask else account})",
+                "inline": False,
+            }
+            print_string = ""
             holdings = brokerObj.get_holdings(key, account)
             if holdings == {}:
-                printAndDiscord("No holdings in Account\n", loop)
+                print_string += "No holdings in Account\n"
             else:
-                print_string = ""
+
                 for stock in holdings:
                     quantity = holdings[stock]["quantity"]
                     price = holdings[stock]["price"]
                     total = holdings[stock]["total"]
                     print_string += f"{stock}: {quantity} @ ${format(price, '0.2f')} = ${format(total, '0.2f')}\n"
-                printAndDiscord(print_string, loop)
-            printAndDiscord(
-                f"Total: ${format(brokerObj.get_account_totals(key, account), '0.2f')}\n",
-                loop,
-            )
-    printAndDiscord("==============================", loop)
+            print_string += f"Total: ${format(brokerObj.get_account_totals(key, account), '0.2f')}\n"
+            field["value"] = print_string
+            EMBED["fields"].append(field)
+    EMBED["description"] = printing
```

Comment:
Embedding works without setting `EMBED["description"] = ""`, so it can be removed along with the printing initialization.